pkgs/development/ocaml-modules/alsa/default.nix

```
@@ -0,0 +1,27 @@
+{ lib, buildDunePackage, fetchFromGitHub, dune-configurator, alsa-lib }:
+
+buildDunePackage rec {
+  pname = "alsa";
+  version = "0.3.0";
+
+  useDune2 = true;
+
+  minimumOCamlVersion = "4.02";
```

```suggestion
  minimalOCamlVersion = "4.02";
```

The `useDune2` attribute is no longer necessary.

---

Thanks!